spec/test-outputs/www-integration.out.vcl

```diff
@@ -426,6 +454,32 @@ sub vcl_deliver {
   }
   # End dynamic section
 
+if (req.http.Cookie ~ "cookies_policy") {
+  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true" && req.http.User-Agent !~ "^GOV\.UK Crawler Worker") {
+    declare local var.variant STRING;
+    set var.variant = req.http.GOVUK-ABTest-StartABusinessSegment;
```

Comment:
To check here, what will the value of `req.http.GOVUK-ABTest-StartABusinessSegment` return if it hasn't been set?

Reply:
Had to set up a fiddle to find out - it's an empty string